index.html

```diff
@@ -9,6 +9,31 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class= "cover-text-container">
+    <img class= "cover-page" 
+    src="https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f17/x/6554b96f839abbe659429a9f8c79115d/book-cover.png" 
+    alt="Wolox books cover page"
+    >
+    <div class= "container-book">
+      <h1 class="title">
+        <strong>Título del libro</strong> 
+        <p class= "genre"> (género)</p>
```

Comment:
I think this should be outside the h1

Reply:
If i put it outside the h1 then the "genre" part is not on the same line as the tittle, genre stays one line below the tittle. Is there a way to do it so they can stay in the same line without having to include the genre inside the h1?